container/webui/Dockerfile

```diff
@@ -9,6 +9,7 @@ RUN zypper ar -p 95 -f http://download.opensuse.org/repositories/devel:openQA/op
     zypper --gpg-auto-import-keys ref && \
     zypper in -y ca-certificates-mozilla curl && \
     zypper in -y --force-resolution openQA-local-db apache2 hostname which w3m && \
+    zypper in -y --force-resolution 'rubygem(sass)' &&\
```

Comment:
It shouldn't be necessary to handle these dependencies here manually. Do we need to handle that better in the spec file?

Reply:
the spec file does not require sass as the assets are precompiled as cache.tar

Reply:
Yes, and hence it shouldn't be required for the container setup as it is using rpms. Tests show that it nevertheless sometimes complains about the missing dependency. I'd say that's actually a bug and this PR is merely a workaround.

Reply:
My point is, if you accept that the workaround is required, then actually better place it where it's broken (in here)

Reply:
Yes, but with a comment explaining the problem and the workaround. If there is "nothing wrong" in the spec file then maybe we can still improve how the situation is handled in our code. As installing in an openSUSE container is for me just one example of what users might do so they could run into the same problem otherwise as well.

Reply:
I add a comment. Is not too much extend because I tried to summarize what you comment here

Reply:
typo, it says `rubygetm:sass`

Reply:
Fixed, I already fixed in my code but I didn't save this version. Thanks